client/extensions/woocommerce/app/settings/payments/payment-method-bacs.js

```diff
@@ -0,0 +1,182 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormLegend from 'components/forms/form-legend';
+import FormTextarea from 'components/forms/form-textarea';
+import FormTextInput from 'components/forms/form-text-input';
+
+class PaymentMethodBACS extends Component {
+	static propTypes = {
+		method: PropTypes.shape( {
+			settings: PropTypes.shape( {
+				title: PropTypes.shape( {
+					id: PropTypes.string.isRequired,
+					label: PropTypes.string.isRequired,
+					type: PropTypes.string.isRequired,
+					value: PropTypes.string.isRequired,
+				} ),
+				accounts: PropTypes.shape( {
+					id: PropTypes.string.isRequired,
+					value: PropTypes.array.isRequired,
+				} ),
+			} ),
+		} ),
+		translate: PropTypes.func.isRequired,
+		onCancel: PropTypes.func.isRequired,
+		onEditField: PropTypes.func.isRequired,
+		onDone: PropTypes.func.isRequired,
+	};
+
+	getAccountData = () => {
+		const { method: { settings } } = this.props;
+
+		// The API _should_ always return this, but just being safe.
+		const accountData = get(
+			settings,
+			[ 'accounts', 'value' ],
+			[
+				{
+					account_name: '',
+					account_number: '',
+					bank_name: '',
+					bic: '',
+					iban: '',
+					sort_code: '',
+				},
+			]
+		);
+
+		return accountData.length ? accountData[ 0 ] : {};
```

Comment:
I take it that having multiple elements in the value array is impossible?

Reply:
Correct in wp-admin you can add multiple accounts - but for simplicity sake in Calypso we are supporting one.

Reply:
Maybe enter that (supporting multiple) as a new low priority issue or a here as a TODO